packages/shopify-codemod/transforms/global-reference-to-import.js

```diff
@@ -146,12 +155,17 @@ export default function globalReferenceToImport(
         });
 
       for (const anImport of imports.values()) {
-        const {file, name} = anImport;
+        const {file, name, hasNamedExports} = anImport;
         const {node: {body}} = path;
+        const importSpecifierBuilder = hasNamedExports ? (
```

Comment:
It's mostly a matter of preference, but personally I don't love switching on which function to eventually invoke, I find `hasNamedExports ? j.importNamespaceSpecifier(identifier) : j.importDefaultSpecifier(identifier)` cleaner.